docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+==== Connector networking configuration
+
+Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+[float]
+[[Preconfigured-opsgenie-configuration]]
+==== Preconfigured connector type
+
+[source,text]
+--
+ my-opsgenie:
+   name: preconfigured-opsgenie-connector-type
+   actionTypeId: .opsgenie
+   config:
+    apiUrl: https://api.opsgenie.com
+   secrets:
+     apiKey: apikey
+--
+
+Config defines information for the connector type.
+
+`apiUrl`:: A string that corresponds to *URL*
+
+Secrets defines sensitive information for the connector type.
+
+`apiKey`:: A string that corresponds to *API Key*.
+
+[float]
+[[define-opsgenie-ui]]
+==== Define connector in Stack Management
+
+Define Opsgenie connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-connector.png[Opsgenie connector]
+
+Test Opsgenie action parameters.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-params-test.png[Opsgenie params test]
+
+[float]
+[[opsgenie-action-configuration]]
+==== Action configuration
+
+The Opsgenie connector supports two types of actions: Creating and Closing alerts. The properties supported for each action are different because Opsgenie defines different properties for each operation.
+
+When testing the Opsgenie connector, choose the appropriate action from the selector. Switching the action will show a different form allowing the specific properties for that type of action to be configured.
+
+Action::    Whether to create or close an alert within Opsgenie. When using the Opsgenie connector for a rule action, the Action selector is hidden and automatically set to the appropriate value. If the action is run when a rule fires, it will create an alert within Opsgenie. If the action is run when a rule recovers, it will close an alert within Opsgenie.
+
+[float]
+[[opsgenie-action-create-alert-configuration]]
+===== Create alert configuration
+
+The create alert action can be configured using the form or a JSON editor.
+
+[float]
+[[opsgenie-action-create-alert-form-configuration]]
+====== Form configuration
+
+The create alert action form has the following configuration properties.
+
+Message::   The message for the alert. This field is required.
+Opsgenie Tags::   The tags for the alert.
+Priority::  The priority level for the alert.
+Description::   The description for the alert to provide detailed information about the alert.
+Alias::   The identifier for the alert. The `Alias` is used for https://support.atlassian.com/opsgenie/docs/what-is-alert-de-duplication/[alert de-duplication].
+Entity::  The domain of the alert.
+Source::  The source of the alert.
+User::    The display name of the owner.
+Note::    Additional information for the alert.
```

Comment:
```suggestion
Note::    Additional information about the alert.
```

Reply:
The main difference here is whether the user is entering additional information that the alert doesn't have, or if they're providing details gathered from the alert to provide more context. If it's the former, ignore this edit.

Reply:
Yeah I think it's the former. The user is creating an alert within opsgenie so they're providing additional information to create the alert.